examples/cdk/functions/common/powertools.ts

```diff
@@ -2,7 +2,7 @@ import { Logger } from '@aws-lambda-powertools/logger';
 import { Metrics } from '@aws-lambda-powertools/metrics';
 import { Tracer } from '@aws-lambda-powertools/tracer';
 
-const awsLambdaPowertoolsVersion = '1.5.0';
+const awsLambdaPowertoolsVersion = '1.11.1';
```

Comment:
We could get the version from `@commons` since we are exporting it for the UA - this way we never have to update this again

Reply:
good point, I'll add the package and export the version in the commons

Reply:
Yes, you are right, you can't copy past and replicate the example any more.